pkg/env/env.go

```diff
@@ -27,6 +27,7 @@ const (
 	FeatureDisableDenyAssignments Feature = iota
 	FeatureDisableSignedCertificates
 	FeatureEnableDevelopmentAuthorizer
+	FeatureRequireD2sV3Workers
```

Comment:
check: the change to dev config generation which adds FeatureRequireD2sV3Workers should also be in this commit